mathesar_ui/src/pages/database/DatabasePage.svelte

```diff
@@ -52,36 +61,97 @@
     targetSchema = schema;
     addEditModal.open();
   }
+
+  function deleteSchema(schema: SchemaEntry) {
```

Comment:
Moved this inside the DatabasePage component so that this component drives all the business logic and child components are only presentational

Reply:
Nice. That is better. Thanks!